snmp/metadata.csv

```diff
@@ -510,6 +510,34 @@ snmp.cwoTfoStatsReservedConn,gauge,,,,"[CISCO-WAN-OPTIMIZATION-MIB] This object
+snmp.cyberpower.ePDU2PhaseStatusPeakLoad,gauge,,,,"[CPS-MIB] The peak current of the Rack PDU phase load in tenths of Amps",0,snmp,,
+snmp.cyberpower.ePDU2PhaseStatusPower,gauge,,,,"[CPS-MIB] Indicates the load power, in watts (kW), consumed on the Rack PDU phase being queried",0,snmp,,
+snmp.cyberpower.ePDU2PhaseStatusPowerFactor,gauge,,,,"[CPS-MIB] Indicates the load power factor, in hundredths, of the Rack PDU phase being queried",0,snmp,,
+snmp.cyberpower.ePDU2PhaseStatusVoltage,gauge,,,,"[CPS-MIB] Indicates the Voltage, in Volts, of the Rack PDU phase being queried",0,snmp,,
+snmp.cyberpower.ePDULoadBankConfig,gauge,,,,"[CPS-MIB] Allows for configuration of each Rack PDU bank. The number of entries is contained in the ePDULoadDevNumBanks OID. (Make 'sum by {X}' queries to count elements with the tag X.)",0,snmp,,
+snmp.cyberpower.ePDULoadStatusActivePower,gauge,,,,"[CPS-MIB] Getting this OID will return the phase/bank active power measured in watts.",0,snmp,,
+snmp.cyberpower.ePDULoadStatusApparentPower,gauge,,,,"[CPS-MIB] Getting this OID will return the phase/bank apparent power measured in VA.",0,snmp,,
+snmp.cyberpower.ePDULoadStatusLoad,gauge,,,,"[CPS-MIB] Getting this OID will return the phase/bank load measured in tenths of Amps.",0,snmp,,
+snmp.cyberpower.ePDULoadStatusPowerFactor,gauge,,,,"[CPS-MIB] Getting this OID will return power factor of the output measured in hundredths. A value of 100 representing a unity power factor (1.00).",0,snmp,,
+snmp.cyberpower.ePDULoadStatusVoltage,gauge,,,,"[CPS-MIB] Getting this OID will return the phase/bank voltage measured in 0.1 volts.",0,snmp,,
+snmp.cyberpower.ePDUOutletStatusActivePower,gauge,,,,"[CPS-MIB] Getting this OID will return the measured Outlet load for an Outlet Monitored Rack PDU in watts",0,snmp,,
+snmp.cyberpower.ePDUOutletStatusLoad,gauge,,,,"[CPS-MIB] Getting this OID will return the measured Outlet load for an Outlet Monitored Rack PDU in tenths of Amps.",0,snmp,,
+snmp.cyberpower.ePDUStatusBank,gauge,,,,"[CPS-MIB] Allows for getting of status of each Rack PDU Bank (Make 'sum by {X}' queries to count elements with the tag X.)",0,snmp,,
+snmp.cyberpower.ePDUStatusInputFrequency,gauge,,,,"[CPS-MIB] The input frequency in 0.1 Hertz.",0,snmp,,
+snmp.cyberpower.ePDUStatusInputVoltage,gauge,,,,"[CPS-MIB] The input voltage in 0.1 Volt.",0,snmp,,
+snmp.cyberpower.ePDUStatusOutlet,gauge,,,,"[CPS-MIB] Allows for getting of status of each Outlet of an Outlet Monitored Rack PDU. (Make 'sum by {X}' queries to count elements with the tag X.)",0,snmp,,
+snmp.cyberpower.ePDUStatusPhase,gauge,,,,"[CPS-MIB] Allows for getting of status of each Rack PDU Phase (Make 'sum by {X}' queries to count elements with the tag X.)",0,snmp,,
+snmp.cyberpower.envirHumidity,gauge,,,,"",0,snmp,,
+snmp.cyberpower.envir_temperature,gauge,,,,"",0,snmp,,
```

Comment:
These two entries are missing descriptions - the MIB descriptions are "The environment relative humidity expressed in percentage." and "The environment temperature expressed in 1/10 Fahrenheit."